winlogbeat/_meta/fields.common.yml

```diff
@@ -22,6 +22,17 @@
   description: >
     Contains data from a Windows event log record.
   fields:
+    - name: user_dbg
+      type: object
+      required: false
+
+    # Candidate to add to ECS?
+    - name: event.code
```

Comment:
Semantics of Windows event ID are useful, they are a log message code, as Andrew explained in his previous comment.

I think it would make sense to have something like this in ECS, even outside of Windows. For example, MySQL uses error codes as well.